src/shogun/classifier/LDA.h

```diff
@@ -158,19 +140,6 @@ class CLDA : public CDenseRealDispatch<CLDA, CLinearMachine>
 			return CT_LDA;
 		}
 
-		/** set features
-		 *
-		 * @param feat features to set
-		 */
-		virtual void set_features(CDotFeatures* feat)
-		{
-			if (feat->get_feature_class() != C_DENSE ||
-				feat->get_feature_type() != F_DREAL)
```

Comment:
these checks are important, need to move them into train if this is removed

Reply:
LDA uses features dispatching, so there is no need for these checks anymore since we are sure that we will get that type of features.